elide-core/src/main/java/com/yahoo/elide/core/filter/Predicate.java

```diff
@@ -43,6 +44,14 @@ public Predicate(PathElement pathElement, Operator op, List<Object> values) {
         this(Collections.singletonList(pathElement), op, values);
     }
 
+    public Predicate(PathElement pathElement, Operator op){
+        this(Collections.singletonList(pathElement), op, new ArrayList<Object>());
```

Comment:
@xiaoyao1991 Yes we do. We have both the `@AllArgsConstructor` for list of `pathElement` and also the non-list version for 3-params constructor. So I follow it to make both versions for 2-params constructor.